tests/tracer/test_utils.py

```diff
@@ -53,78 +52,19 @@ def test_asbool(self):
         ("key: val", dict(key=" val"), None),
         ("key key: val", {"key key": " val"}, None),
         ("key: val,key2:val2", dict(key=" val", key2="val2"), None),
-        (" key: val,key2:val2", {" key": " val", "key2": "val2"}, None),
+        (" key: val,key2:val2", {"key": " val", "key2": "val2"}, None),
```

Comment:
We should document this in the release note in case someone runs into this issue

Reply:
Just to clarify, are you referring to whitespace/separator stripping of the tags string?

Reply:
@P403n1x87 can we still do this (also for other cases we know of)?

Reply:
This is an interesting one. I think we document the tags as starting with a letter, so the previous behaviour was wrong. https://docs.datadoghq.com/getting_started/tagging/

Reply:
Yeah, even though the previous was wrong I'd like to add it to the release note so that if someone has weird behaviour because their DD_TAGS break they can go to the release note and get a hint to what the issue might be.